src/components/02_atoms/Widgets/FileUploadWidget.js

```diff
@@ -60,7 +60,7 @@ const FileUploadWidget = ({
 
   const items = getItemsAsArray(multiple, value.data)
     // Default schema creates stub entries, which we don't need here.
-    .filter(item => item.id);
+    .filter(item => item.file.id);
```

Comment:
Are you sure about this change? There should be an id

Reply:
Sure ? No.. but I think maybe yes and we need to adjust our code.

 I am still looking into .. What I am trying to understand/correct is :

If you look at the storybook definition ( showing the Widget with multiple files already 'added' )

items is an array of objects with the following structure { file: {} meta{} } 

Currently when the FileUploadWidget is constructed as part of the recipe form just the 'file' object is passed in as item(s)

It makes sense to me that with say 3 files uploaded there are 3 distinct alternate text string to process/display

What I need to take care of an get right is how 'file' and 'meta' are iterated over and transported through the render function and how the defaults are initialized/derived from the schema.

<FileUploadWidget value={data: [ 
      {file:{/*data struct */}, meta: {/* data struct */}}
      {file:{/*data struct */}, meta: {/* data struct */}}
      ...
]} /> 

Anyway I hope this is a better explanation of my current mental block.